recipes/sfdmap/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "sfdmap" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "12d9d03ab5f32a70ce363543fae53b568ae457dfcd8e6045f7ab5142a59a06a2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - sfdmap
+
+about:
+  home: https://github.com/kbarbary/sfdmap
+  license: MIT
+  license_file: licenses/LICENSE
+  summary: 'Get E(B-V) values from the Schlegel et al (1998) galactic dust map'
+  description: |
+    A minimal, fast, MIT-licensed Python module for getting E(B-V)
+    values from Schlegel, Finkbeiner & Davis (1998) galactic dust map FITS
+    files. The data files must be downloaded separately. A FITS reader
+    (either fitsio or astropy) is required. 
+  doc_url: http://github.com/kbarbary/sfdmap
+  dev_url: https://github.com/kbarbary/sfdmap
+
+extra:
+  recipe-maintainers:
+    - kbarbary
+    - mwcraig
```

Comment:
@mwcraig -- Are you ok being added as a co-maintainer on the sfdmap package on conda-forge?